bots/images/scripts/fedora-testing.setup

```diff
@@ -31,6 +31,7 @@ libssh \
 libvirt \
 libvirt-client \
 NetworkManager-team \
+PackageKit \
```

Comment:
fedora-testing rebuilds have some consistent OpenShift failures (#6745) which need to be investigated first. As we don't run fedora-testing in CI regularly, I thought I could rebuild it separately?